nvim-plugin/README.md

```diff
@@ -31,9 +31,10 @@ Usually, you will add the string `"ethersync/ethersync-nvim"` to your plugin man
 {
   "ethersync/ethersync-nvim",
   keys = { 
-    { "<leader>j", "<cmd>EthersyncJumpToCursor<cr>" },
-    { "<leader>f", "<cmd>EthersyncFollow<cr>" },
-    { "<leader>u", "<cmd>EthersyncStopFollow<cr>" },
+    { "<leader>ej", "<cmd>EthersyncJumpToCursor<cr>" },
+    { "<leader>ef", "<cmd>EthersyncFollow<cr>" },
+    { "<leader>eu", "<cmd>EthersyncUnfollow<cr>" },
```

Comment:
My feeling is: If we have the <esc> mapping, we don't even need to suggest the <leader>eu one. Less mappings to keep track of.